packages/fluent/scss/checkbox/_layout.scss

```diff
@@ -0,0 +1,248 @@
+        }
+    }
+
+    // Indeterminate state
+    .k-checkbox:indeterminate,
+    .k-checkbox.k-indeterminate {
+        &::after {
+            content: "";
+            width: var( --INTERNAL--kendo-checkbox-indeterminate-width, #{ $kendo-checkbox-indeterminate-width });
+            height: var( --INTERNAL--kendo-checkbox-indeterminate-height, #{ $kendo-checkbox-indeterminate-height });
+            border-width: var( --INTERNAL--kendo-checkbox-indeterminate-border-width, 5px);
+            border-style: solid;
+            border-radius: 2px;
+            box-sizing: border-box;
+            position: absolute;
+            top: var( --INTERNAL--kendo-checkbox-indeterminate-top, #{ $kendo-checkbox-indeterminate-top });
+            left: var( --INTERNAL--kendo-checkbox-indeterminate-left, #{ $kendo-checkbox-indeterminate-left });
+        }
+    }
+
+    // Checkbox wrap
+    .k-checkbox-wrap {
+        flex: none;
+        display: inline-flex;
+        flex-flow: row nowrap;
+        gap: 0;
+        align-items: center;
+        align-self: flex-start;
+        vertical-align: middle;
+        position: relative;
+
+        &::before {
+            content: "\200b";
+            width: 0px;
+            overflow: hidden;
+            flex: none;
+            display: inline-block;
+            vertical-align: top;
+        }
+    }
+
+
+    // Checkbox label
+    .k-checkbox-label {
+        margin: 0;
+        padding: 0;
+        display: inline-flex;
+        align-items: flex-start;
+        gap: var( --kendo-checkbox-label-margin-x, #{ $kendo-checkbox-label-margin-x } );
+        vertical-align: middle;
+        position: relative;
+        cursor: pointer;
+
+        .k-ripple {
+            // Hide ripple temporarily
+            visibility: hidden !important; // sass-lint:disable-line no-important
+        }
+    }
+    .k-checkbox + .k-label,
+    .k-checkbox-wrap + .k-label,
+    .k-checkbox + .k-checkbox-label,
+    .k-checkbox-wrap + .k-checkbox-label {
+        display: inline;
+        margin-inline-start: var( --kendo-checkbox-label-margin-x, #{ $kendo-checkbox-label-margin-x } );
+    }
+
+
+    // Empty label
+    .k-checkbox-label:empty {
+        display: none !important; // sass-lint:disable-line no-important
+    }
+    .k-checkbox-label.k-no-text {
+        min-width: 1px;
+    }
+
+
+    // Checkbox list
+    .k-checkbox-list {
+        margin: 0;
+        padding: 0;
+        display: flex;
+        flex-flow: column nowrap;
+        gap: 0;
+        list-style: none;
+    }
+    .k-checkbox-item,
+    .k-checkbox-list-item {
+        padding-inline: var( --kendo-checkbox-list-item-padding-x, #{ $kendo-checkbox-list-item-padding-x } );
+        padding-block: var( --kendo-checkbox-list-item-padding-y, #{ $kendo-checkbox-list-item-padding-y } );
+        display: flex;
+        flex-flow: row nowrap;
+        align-items: center;
+        align-content: center;
+        gap: map-get( $kendo-spacing, 1 );
```

Comment:
check if this spacing is accurate, by design I think it's 8px, which means kendo-spacing, 2